jenkins/scripts/dynamic_worker_workflow/feature_tests.sh

```diff
@@ -68,6 +68,8 @@ if [[ "${UPDATED_REPO}" != *"${REPO_ORG}/${REPO_NAME}"* ]] ||
     git config user.name "Test"
     git remote add test "${UPDATED_REPO}"
     git fetch test
+    # if triggered from prow we cannot get the ghprbAuthorRepoGitUrl then we pull the PR
+    git fetch origin "pull/${PR_ID:-0}/head:${UPDATED_BRANCH}-branch" || true
```

Comment:
I don't like the idea of doing invalid git fetches, can we put this in conditional so we'd only pull when we have `$PR_ID`?

Reply:
Here we are defaulting the var to 0 if null then should we just check if the PR_ID is not zero to run this ?

Reply:
PR number 0 makes it invalid to GH. We're calling external service with invalid ID, hence skipping it if PR_ID is 0 would be preferable.